src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -132,6 +132,11 @@ class NoTemplatesServiceApi(object):
         and trailing
         whitespace.
 
+        Example:
+          >>> from google.example.v1 import no_path_templates_pb2
+          >>> api = NoTemplatesServiceApi()
```

Comment:
@tcoffee-google I looked briefly at how you're using PythonImportHandler in discovery sample gen, and it doesn't look like you're using it for the same use case as what I'm doing here: importing just the API class itself.